src/electionguard/voting/message_reps.h

```diff
@@ -17,4 +14,9 @@ struct ballot_identifier_rep
     uint64_t id;
 };
 
+struct external_identifer_rep
```

Comment:
either remove this struct or add the external identifier to the serialization process